scripts/events.py

```diff
@@ -2129,7 +2129,29 @@ def handle_mass_extinctions(self, cat):  # pylint: disable=unused-argument
             for poor_little_meowmeow in dead_cats:
                 poor_little_meowmeow.die()
                 # this next bit is temporary until we can rework it
-                History.add_death(poor_little_meowmeow, 'This cat died after disaster struck the Clan.')
+                import re
```

Comment:
imports should be done in the first lines of a file (cleaner structure) just put it at the top to the other imports \info

Reply:
Done!